src/shim/telemetryClient.ts

```diff
@@ -436,4 +712,16 @@ export class TelemetryClient {
         const record = this._telemetryToLogRecord(telemetry, baseType, baseData);
         return record;
     }
+
+    public trackNodeHttpRequestSync(telemetry: Contracts.NodeHttpRequestTelemetry) {
+        Logger.getInstance().warn("trackNodeHttpRequestSync is not implemented and is a no-op.");
+    }
+
+    public trackNodeHttpRequest(telemetry: Contracts.NodeHttpRequestTelemetry) {
+        Logger.getInstance().warn("trackNodeHttpRequest is not implemented and is a no-op.");
```

Comment:
Why are these not supported? what exactly these were doing before?

Reply:
They were some helper functions that called methods (`trackRequestSync` and `trackRequest`) inside of the `AutoCollectHttpRequests` class. Given that OpenTelemetry tracks requests synchronously by default I figured it'd be less confusing to just say these methods weren't supported and have users unify on calling the single `trackRequest` method. 

I think it'd also be fair to implement them all calling the `TelemetryClient.trackRequest()` or `trackDependency` methods and just note that these will all be synchronous.

Reply:
I'll update the message here to make it more explicit to users the expected alternative method to call.